### PR TITLE
feat:create helpers

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyenthimy-agilityio @hungnguyenhuu-agilityio Please help me to check and merge it

### CODE REVIEW

File: src/javascripts/helpers/apiRequest.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class ApiRequest {
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/helpers/apiRequest.js]: This one is not a helper. Please consider to restructure it

Comment (LeVHoangduc) [reply src/javascripts/helpers/apiRequest.js]: @hungnguyenhuu-agilityio I have updated it

File: src/javascripts/helpers/formValidator.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import { MESSAGE, REGEX } from "../constants/constants";
+
+/**
+ * Function for check if the form is valid
+ * @param {Object} form is form validation
+ * @param {Object} fields is field validation data
+ * @param {Object} object is object validation
+ * @returns {Boolean}
+ */
+const validationFields = (form, fields, object) => {
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/helpers/formValidator.js:10]: With the return boolean, the naming should be isValidFields, is...

File: src/javascripts/helpers/formValidator.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      inputEl.classList.add("error");
+      errorEl.textContent = field.invalidMessage;
+      errorEl.classList.add("active");
+      isValid = false;
+    }
+    // If the field is valid, remove any error styling and message
+    else {
+      inputEl.classList.remove("error");
+      errorEl.textContent = "";
+      errorEl.classList.remove("active");
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/helpers/formValidator.js:36]:
```     
 inputEl.classList.remove("error");
      errorEl.textContent = "";
      errorEl.classList.remove("active");
```
     
this one should code on view. Not here

File: src/javascripts/helpers/formValidator.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  return isValid;
+};
+
+/**
+ * Form validator check if the information is valid.
+ * @param {string} validation identify the field
+ * @param {object} object is object validation
+ * @returns {Boolean} is the form valid.
+ */
+export const formValidator = (validation) => (object) => {
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/helpers/formValidator.js:48]: Your helper is not good. Please update again

**Pull request closed without merging**